src/Components/Toolbar/SaveCharacter/SaveAlerts/displayAlert.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useAppSelector } from 'features/reduxHooks';
+import { Alert } from 'react-bootstrap';
+
+export const DisplayAlert: React.FC<{ alertType: string }> = ({ alertType }) => {
+  const characterName = useAppSelector((state) => state.character.name);
+  const nameText = characterName;
+  const [show, setShow] = useState(true);
+
+  const createAlert = (variant: string, text: string) => {
+    if (show) {
+      return (
+        <div className="save-alert">
+          <Alert variant={variant} onClose={() => setShow(false)} dismissible>
+            <Alert.Heading>{text}</Alert.Heading>
+            <div className="d-flex justify-content-end"></div>
+          </Alert>
+        </div>
+      );
+    } else return;
+  };
+  //check below
+  const alert = alertType;
+  let saveAlert: JSX.Element | undefined;
+  switch (alert) {
+    case 'success':
```

Comment:
if you were gonna stick with this, create alert is really returning a React component, id rename createAlert to Alert then
```diff
    case 'success':
-      saveAlert = createAlert('success', `${nameText} has been succesfully saved.`);
-       break;
+     return <Alert variant='success' text=`${nameText} has been succesfully saved.` />
```